libraries/AP_Arming/AP_Arming.h

```diff
@@ -166,6 +174,21 @@ class AP_Arming {
         MIS_ITEM_CHECK_RALLY         = (1 << 5),
         MIS_ITEM_CHECK_MAX
     };
+
+    // auxiliary authorisation
+    static const uint8_t aux_auth_count_max = 3;    // maximum number of auxiliary authorisers
+    static const uint8_t aux_auth_str_len = 42;     // maximum length of failure message (50-8 for "PreArm: ")
+    enum class AuxAuthStates : uint8_t {
+        AUX_AUTH_STATE_UNASSIGNED = 0,
```

Comment:
This state doesn't matter, you assign them in a strictly increasing order based on `auth_count` which means any ID less then `auth_count` *must* be in the NO_RESPONSE, FAILED, or PASSED state.

Reply:
Also with an enum class like this we shouldn't be prefixing `AUX_AUTH_STATE` to all the members. It's made all the users incredibly verbose and less clear whats happening.

Reply:
ok, removed the no-response and shortened the names of the states to NO_RESPONSE, AUTH_FAILED and AUTH_PASSED